posts/2024-02-08-interviews-anirban_chetia/index.qmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "data.table interviews!"
+author: "Anirban Chetia"
+date: "2024-02-10"
+categories: [interviews]
+image: ""
+---
+
+## What's it all about?
+
+As part of the project affiliated with [NSF POSE](https://new.nsf.gov/funding/opportunities/pathways-enable-open-source-ecosystems-pose) and based on expanding the open-source ecosystem around `data.table`, several interviews had to be conducted under [NSF's I-CORPS](https://new.nsf.gov/funding/initiatives/i-corps) program (Winter 2024 Cohort) to gather information as to how `data.table` as an open-source project can improve upon, along with ways to discover the project's sustainability. The timeline for the interviews was originally from January to March '24 with a required interview count of 100, but the program downscaled in time (17th of January to four weeks after - just wrapped up my final week as I'm writing this!) and so did the count, to a total of 60. [Issue#5880](https://github.com/Rdatatable/data.table/issues/5880) mentions this, and has a link to the Google Doc that contains the list of people interviewed.
```

Comment:
can improve (remove upon)

can you please break this up into 2+ sentences? The timeline for the interviews was originally from January to March '24 with a required interview count of 100, but the program downscaled in time (17th of January to four weeks after - just wrapped up my final week as I'm writing this!) and so did the count, to a total of 60.